scenes/tests/3d-view.tscn

```diff
@@ -1,17 +1,33 @@
-[gd_scene load_steps=3 format=2]
+<<<<<<< HEAD
```

Comment:
Merge Conflict boy, please resolve
contact if you need help

Reply:
Oh yeah, miss that one